src/base/USong.pas

```diff
@@ -653,24 +650,14 @@ function TSong.LoadOpenedSong(SongFile: TTextFileStream; FileNamePath: IPath; Du
           end;
 
           // add notes
-          if (CurrentTrack <> 2) then
-          begin
-            // P1
-            if (Tracks[CurrentTrack].High < 0) or (Tracks[CurrentTrack].High > 5000) then
-            begin
-              Log.LogError('Found faulty song. Did you forget a P1 or P2 tag? "'+Param0+' '+IntToStr(Param1)+
-              ' '+IntToStr(Param2)+' '+IntToStr(Param3)+ParamLyric+'" -> '+
-              FileNamePath.ToNative+' Line:'+IntToStr(FileLineNo));
-              Break;
-            end;
-            ParseNote(CurrentTrack, Param0, (Param1+Rel[CurrentTrack]) * Mult, Param2 * Mult, Param3, ParamLyric);
-          end
-          else
+          if (Tracks[CurrentTrack].High < 0) or (Tracks[CurrentTrack].High > 5000) then
```

Comment:
why this `Tracks[CurrentTrack].High > 5000`?

Reply:
I don't know. It was there already before I did anything. I wonder if it's a limit on 5000 sentences or something?
I think I kept the behaviour of whatever this was doing in the past, and was going to look into what this 5000 number was after this PR, but I can also look into what it does as a further reply.